climada/engine/impact.py

```diff
@@ -842,11 +859,12 @@ def write_col(i_col, imp_ws, xls_data):
         write_col(7, imp_ws, self.event_name)
         write_col(8, imp_ws, self.date)
         write_col(9, imp_ws, self.frequency)
-        write_col(10, imp_ws, self.at_event)
-        write_col(11, imp_ws, self.eai_exp)
-        write_col(12, imp_ws, self.coord_exp[:, 0])
-        write_col(13, imp_ws, self.coord_exp[:, 1])
-        write_col(14, imp_ws, [str(self.crs)])
+        write_col(10, imp_ws, [self.frequency_unit])
```

Comment:
Would it not be good to have the frequency unit at the end, such that files that have been created so far are still readable by custom scripts?

Reply:
Ok, fine for me.